actions/admin/upgrades/upgrade_friends_private_acls.php

```diff
@@ -24,7 +24,8 @@
 
 // Offset is the total amount of processed users so far.
 $offset = (int) get_input('processed', 0);
-$limit = 50;
+$class = get_input('class');
```

Comment:
This file should be renamed ```actions/admin/upgrade.php```. The individual upgrade it calls is then defined by this parameter.

I'll rename this file in a minute...